package.json

```diff
@@ -24,15 +25,16 @@
     "sign": "npm run clean:extension && npm run build:extension && web-ext sign --source-dir=build/extension && mv web-ext-artifacts/*.xpi ./addon.xpi",
     "run": "npm run build:extension && web-ext run --source-dir=build/extension",
     "lint": "eslint --color src",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "npm run lint",
     "deploy": "gh-pages -d build/web",
     "release:base": "npm run clean && npm run build:web && npm run sign && mv addon.xpi build/web && npm run deploy",
```

Comment:
Wasn't sure about this... I was tempted to try using `npm-run-all clean build:web sign` to be a bit less verbose. 🤷‍♀️